case_studies/weather_dependent_app_example/README.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+This is an example Splunk App to demonstrate how to use setup pages that adhere to Splunk Application Certification standards.
```

Comment:
I'd add a bit more context here:

"This is an example Splunk App that demonstrates how to use a setup page to check for the installation of app dependencies. This example adheres to Splunk Application Certification standards.

This app requires that users install the weather app as a dependency in order to complete the setup process. The setup page checks whether the dependent app is installed before allowing users to access the app home page."

Reply:
Fixed.